src/PIL/ImageMode.py

```diff
@@ -23,18 +23,25 @@
 class ModeDescriptor:
     """Wrapper for mode strings."""
 
-    def __init__(self, mode, bands, basemode, basetype, typestr):
+    def __init__(
+        self,
+        mode: str,
+        bands: str | tuple[str, ...],
+        basemode: str,
+        basetype: str | tuple[str, ...],
```

Comment:
```suggestion
        basetype: str,
```
I don't think `basetype` is ever a tuple.